airflow/docker_image.go

```diff
@@ -140,7 +147,18 @@ func (d *DockerImage) Pytest(pytestFile, airflowHome, envFile string, pytestArgs
 	if err != nil {
 		log.Debug(err)
 	}
-
+	if htmlReport {
+		// Copy the dag-test-report.html file from the container to the destination folder
+		err = cmdExec(dockerCommand, nil, stderr, "cp", "astro-pytest:/usr/local/airflow/dag-test-report.html", "./"+testHomeDirectory)
+		if err != nil {
+			// Remove the temporary container
+			err = cmdExec(dockerCommand, nil, stderr, "rm", "astro-pytest")
```

Comment:
nit: we should move deleting pytest container to defer statement right after we create it